conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -230,6 +230,10 @@ func checkRounds(logger *logrus.Logger, catchpointRound, nodeRound, targetRound
 
 func (algodImp *algodImporter) needsCatchup(targetRound uint64) bool {
 	if algodImp.mode == followerMode {
+		if targetRound == 0 {
+			algodImp.logger.Info("No state deltas are ever available for round 0")
+			return true
+		}
```

Comment:
Why does this return true? If the target round is 0 I think it would go ahead and no-op in the catchup function, but is it needed? Maybe in the case where Conduit is fully caught up and it is told to go back to 0?

I think this would be slightly more correct so that it still checks for the block.
```
	if algodImp.mode == followerMode && targetRound != 0 {
```

Or maybe even
```
if targetRound == 0 {
    targetRound++
}
```

Reply:
cc: @Eric-Warehime

Reply:
There were only two choices of what to return and that seemed the most reasonable. But we could change the API,.